### DISCUSSION TITLE
About likelihood

### DISCUSSION THREAD

**Discussion by hahia** [theislab/scvelo#467] [Q&A]
Hello
I have a naive question.
what the **likelihood** of gene mean?

and how is it caculated? Is the distance from the data point to the fitted curve? 
Thank any comments.

**Comment by WeilerP** [comment]
The likelihood is a measure of goodness of fit. I'll denote the likelihood function for given parameter `p` and data `X` by `L(p | X)`. Consider now some given data `X` and two sets of parameters `p_1` and `p_2`. `L(p_1 | X) > L(p_1 | X)` implies that the data is more likely to have been generated by the paramter set `p_1`.
The likelihood is related to the distance between the actual data point and fitted curve. It's definition depends on the assumed underlying probability distribution. In our case, we consider Laplace distributed data.
